thunder/core/proxies.py

```diff
@@ -129,9 +129,22 @@ def __init__(self, name: str | None = None, *, prefix: None | str = None, histor
     def name(self) -> str:
         return self._name
 
+    def replace(self, **changes):
+        r"""Return a copy of the Proxy object with new values for the specified fields as given to the constructor as arguments.
+        Valid kewword arguments are `name`, ``history`.
```

Comment:
```suggestion
        Valid keyword arguments are ``"name"``, ``"history"``.
```

Reply:
I think it's with double backtics per https://bashtage.github.io/sphinx-material/rst-cheatsheet/rst-cheatsheet.html